packages/fury-adapter-oas3-parser/lib/parser/oas/parseSchemaObject.js

```diff
@@ -131,8 +131,13 @@ function validateValuesMatchSchema(context, schema) {
 function parseSchema(context) {
   const { namespace } = context;
 
-  const ensureValidType = R.unless(isValidType, createWarning(namespace,
-    `'Schema Object' 'type' must be either ${types.join(', ')}`));
+  const ensureValidType = R.unless(
+    isValidType,
+    R.compose(
+      createWarning(namespace, `'Schema Object' 'type' must be either ${types.join(', ')}`),
+      getValue
```

Comment:
"type" the element passed ot createWarning is a member element and not a value and thus does not contain source maps.